<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>电机控制</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextBrowser" name="dataShowTB">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>371</width>
      <height>421</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>UpArrowCursor</cursorShape>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空数据</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendLineEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>480</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>321</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="iLabel">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>I</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="iLineEdit"/>
     <widget class="QPushButton" name="iBtn">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="dLabel">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dLineEdit"/>
     <widget class="QPushButton" name="dBtn">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="pLabel">
      <property name="font">
       <font>
        <family>AcadEref</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>P</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pLineEdit"/>
     <widget class="QPushButton" name="pBtn">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="openBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRateCbx">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="currentText">
      <string>1200</string>
     </property>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="serialPortCbx">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="serialPortLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
    <widget class="QLabel" name="baudRateLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>AcadEref</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="speedLabel">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>速度</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="speedLiineEdit"/>
     <widget class="QPushButton" name="speedBtn">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>784</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
